doc/rose-rug-task-run.html

```diff
@@ -896,6 +896,17 @@ <h2 id="rose-task-run.util.rose_prune">Built-in Application:
     <p>The following settings are accepted:</p>
 
     <dl>
+      <dt><kbd>cycle-format{key}=format</kbd></dt>
+
+      <dd>Specify a key to a format string for use in conjunction with a
+      <kbd>prune{item-root}=cycle:globs ...</kbd> setting. For example, we may
+      have something like <kbd>cycle-format{cycle_year}=CCYY</kbd> and
+      <kbd>prune{share}=-P1Y:xmas-present-%(cycle_year)s/</kbd>. The
```

Comment:
The other examples then follow the example with what the example would clear out.